src/test/java/com/hillel/items_exchange/controller/UserControllerTest.java

```diff
@@ -89,7 +89,7 @@ void negativeTestReceivingInformationAboutAnotherUser() throws Exception {
     @Test
     @WithMockUser(username = "admin")
     @Transactional
-    @DataSet("database_init.yml")
+    @DataSet("user/update_init.yml")
```

Comment:
Don't do that. It's end2end test so we need to be sure that it's work with completely filled database. For validation use ExpectedDataset here as well.
Update all such methods here.